diff --git a/doc/meson.build b/doc/meson.build
index f261344..8053b0b 100644
--- a/doc/meson.build
+++ b/doc/meson.build
@@ -8,7 +8,7 @@ sphinx_build_dir = meson.current_build_dir()
 
 # Build documentation
 foreach b: doc_builders
-  run_command(sphinx, '-b', b, sphinx_source_dir, join_paths(sphinx_build_dir, b))
+  run_command(sphinx, '-b', b, sphinx_source_dir, join_paths(sphinx_build_dir, b), '-d', join_paths(sphinx_build_dir, '.doctrees'))
 endforeach
 
 # `in` operator requires meson 0.49.0
diff --git a/meson_options.txt b/meson_options.txt
index dc6690b..f0a665c 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -1 +1 @@
-option('doc_builders', type : 'array', choices : ['html', 'man'], value : ['html', 'man'])
+option('doc_builders', type : 'array', choices : ['html', 'man'], value : ['html'])
